src/applications/discharge-wizard/constants/display-conditions.js

```diff
@@ -29,25 +29,23 @@ const {
   REASON_8,
   REASON_6,
   REASON_7,
-  COURT_MARTIAL_1,
-  COURT_MARTIAL_2,
-  COURT_MARTIAL_3,
-  INTENTION_2,
-  PREV_APPLICATION_1,
-  PREV_APPLICATION_2,
-  //   DISCHARGE_TYPE_1,
-  DISCHARGE_TYPE_2,
-  PREV_APPLICATION_YEAR_1A,
-  PREV_APPLICATION_YEAR_1B,
-  PREV_APPLICATION_YEAR_1C,
-  PREV_APPLICATION_YEAR_2A,
-  PREV_APPLICATION_YEAR_2B,
-  PREV_APPLICATION_YEAR_2C,
-  PREV_APPLICATION_TYPE_3A,
-  PREV_APPLICATION_TYPE_3B,
-  PREV_APPLICATION_TYPE_1,
-  PREV_APPLICATION_TYPE_2,
-  PREV_APPLICATION_TYPE_4,
+  COURT_MARTIAL_YES,
+  COURT_MARTIAL_NO,
+  NOT_SURE,
+  INTENTION_NO,
+  YES,
+  NO,
+  DISCHARGE_DISHONORABLE,
+  PREV_APPLICATION_BEFORE_2014,
+  PREV_APPLICATION_BEFORE_2011,
+  PREV_APPLICATION_BEFORE_2017,
+  PREV_APPLICATION_AFTER_2014,
+  PREV_APPLICATION_AFTER_2011,
+  PREV_APPLICATION_AFTER_2017,
+  PREV_APPLICATION_BCMR,
+  PREV_APPLICATION_BCNR,
+  PREV_APPLICATION_DRB_DOCUMENTARY,
+  PREV_APPLICATION_DRB_PERSONAL,
   FAILURE_TO_EXHAUST_1,
```

Comment:
Do the `FAILURE_TO_EXHAUST_1` and `FAILURE_TO_EXHAUST_2` consts need to be made more specific?

Reply:
I did not catch this thanks